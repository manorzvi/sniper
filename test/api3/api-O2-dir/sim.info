{'cmdline': ['../../run-sniper',
             '-n',
             '1',
             '-d',
             'api-O2-dir',
             '--',
             './api-O2'],
 'git_revision': '',
 'host': 'butler',
 'pin_version': 'unknown',
 'rusage': (25.049229,
            0.100826,
            38364,
            0,
            0,
            0,
            8033,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            48545,
            41),
 'snipercmd': ['bash',
               '-c',
               '/home/manorz/sniper-7.4/lib/sniper -c /home/manorz/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/manorz/sniper-7.4/test/api3/api-O2-dir --config=/home/manorz/sniper-7.4/config/nehalem.cfg --config=/home/manorz/sniper-7.4/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpyunpJM/run_benchmarks'],
 't_elapsed': 31.094788074493408,
 't_start': 1633352583.413154,
 'tracecmds': [['/home/manorz/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpyunpJM/run_benchmarks',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api-O2']],
 'user': 'root',
 'vmem': 220901376L}
