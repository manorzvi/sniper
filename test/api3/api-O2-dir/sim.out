                                     | Core 0    
  Instructions                       |   40121623
  Cycles                             |   13725068
  IPC                                |       2.92
  Time (ns)                          |    5159800
  Idle time (ns)                     |         39
  Idle time (%)                      |       0.0%
Branch predictor stats               |           
  num correct                        |   10022006
  num incorrect                      |       1519
  misprediction rate                 |      0.02%
  mpki                               |       0.04
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |   20015337
    num misses                       |         58
    miss rate                        |      0.00%
    mpki                             |       0.00
  D-TLB                              |           
    num accesses                     |      40280
    num misses                       |         79
    miss rate                        |      0.20%
    mpki                             |       0.00
  L2 TLB                             |           
    num accesses                     |        137
    num misses                       |        128
    miss rate                        |     93.43%
    mpki                             |       0.00
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |   20015337
    num cache misses                 |        892
    miss rate                        |      0.00%
    mpki                             |       0.02
  Cache L1-D                         |           
    num cache accesses               |      40280
    num cache misses                 |       1805
    miss rate                        |      4.48%
    mpki                             |       0.04
  Cache L2                           |           
    num cache accesses               |       2734
    num cache misses                 |       2487
    miss rate                        |     90.97%
    mpki                             |       0.06
  Cache L3                           |           
    num cache accesses               |       2526
    num cache misses                 |       2481
    miss rate                        |     98.22%
    mpki                             |       0.06
DRAM summary                         |           
  num dram accesses                  |       2481
  average dram access latency (ns)   |      72.85
  average dram queueing delay        |      19.42
  average dram bandwidth utilization |      0.00%
Coherency Traffic                    |           
  num loads from dram                |       1114
  num loads from dram cache          |          0
  num loads from remote cache        |          0
